Projects/05matchGame/index.js

```diff
@@ -0,0 +1,402 @@
+  let currentMatchedCards = 0;
+  let result = "";
+  let currentTime = 0;
+
+  let uiElements = {
+    $containerCards: document.querySelector(".containerCards"),
+    $popupContainer: document.querySelector(".popup"),
+    $closePopup: document.querySelector(".close"),
+    $faceCard: document.querySelectorAll(".faceCard"),
+    $flipCounter: document.querySelector(".flipCounter"),
+    $backCard: document.querySelector(".backCard"),
+    $timeCounter: document.querySelector(".timeCounter"),
+    $resetBtn: document.querySelector(".resetBtn"),
+    $gameResults: document.querySelector(".gameResults"),
+    $secondCounter: document.querySelector(".counterSeconds"),
+    $numbersOfFlips: document.querySelector(".counterFlips"),
+    $heightScore: document.querySelector(".heightScoreMsg"),
+  };
+
+  let sysMsg = {
+    popup: document.querySelector(".popup"),
+    popupWinLoseNotResponse: document.querySelector(".sysMsg"),
+    popupImage: document.querySelector(".popupContent"),
+    resultGameOver: "Game Over",
+    resultWin: "Win",
+    notResponseAPI: "Error: Site not found",
+  };
+
+  //check the screen size - the game starts only for screens 700px+;
+  if (isNotMobile) {
+    //init game and start first game;
+    await handleRequestDeckId();
+    await handleRequestCards();
+    await handleStartingGame();
+
+    //get deckId from API;
+    async function handleRequestDeckId() {
+      return fetch(
+        `https://deckofcardsapi.com/api/deck/new/shuffle/?deck_count=1`
+      )
+        .then((response) => response.json())
+        .then((data) => {
+          deckId = data.deck_id;
+        })
+        .catch(() => {
+          notResponseApi();
+        });
+    }
+
+    //get cards from API;
+    async function handleRequestCards() {
+      return fetch(
+        `https://deckofcardsapi.com/api/deck/${deckId}/draw/?count=8`
+      )
+        .then((response) => response.json())
+        .then((data) => {
+          savedCards.push(data.cards);
+          let mergeSavedCards = savedCards.flat(1);
+
+          //copy the 8 cards from API - make an array with 16 cards - 8 pairs of cards;
+          let cloneSavedCards = [...mergeSavedCards];
+
+          allFaceCards = mergeSavedCards.concat(cloneSavedCards);
+          allFaceCards
+            .map((cardFace) => {
+              allSavedCards.push(cardFace.images.png);
+
+              //randomize the cards from API;
+              return allSavedCards.sort(() => Math.random() - 0.5);
+            })
+            .join("");
+        })
+        .catch(() => {
+          notResponseApi();
+        });
+    }
+
+    //show popup with sysMsg - not response from API;
+    function notResponseApi() {
+      document.querySelector(".pageGame").innerHTML = "";
+      uiElements.$popupContainer.style.display = "block";
+      sysMsg.popupWinLoseNotResponse.innerHTML = sysMsg.notResponseAPI;
+      sysMsg.popupWinLoseNotResponse.classList.add("centerSysMsg");
```

Comment:
create new object with name of classes and use it instead of direct typing here in the code (search in your code for "classList"), example of object:
`const cssClass = {
centerSysMsg: 'centerSysMsg',
...
}`

Reply:
@kvmladenov  done